### PR TITLE
Adding bridg pathways to deployer options

### PR DESCRIPTION
**Pull request by Dearnshaw-VA**
Updated choices in JenkinsFile to include bridg-pathways, added bridg-pathways.conf and .yaml. I based these off of the carma ones.

### CODE REVIEW

File: products/bridg-pathways.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DU_ARTIFACT=health-apis-bridg-pathways-deployment
+DU_VERSION=1.0.42
```
Comment (ilaflamme-va) [inline products/bridg-pathways.conf]: DU_VERSION=1.0.42? Isn't this new?

File: products/bridg-pathways.conf [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+DU_NAMESPACE=bridg-pathways
+DU_DECRYPTION_KEY="$DEPLOYMENT_CRYPTO_KEY"
```
Comment (ilaflamme-va) [inline products/bridg-pathways.conf]: Will you be using the same crypto-key for bridg?

File: products/bridg-pathways.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    deployment-id: $K8S_DEPLOYMENT_ID
+    deployment-unit: $PRODUCT
+    deployment-unit-artifact: $DU_ARTIFACT
+    deployment-unit-version: $DU_VERSION
+    deployment-date: $BUILD_DATE
+    deployment-s3-bucket: $DU_AWS_BUCKET_NAME
+    deployment-s3-folder: $DU_S3_FOLDER
+    deployment-app-version: $CS_VERSION
+    deployment-test-status: UNTESTED
+---
```
Comment (ilaflamme-va) [inline products/bridg-pathways.yaml]: The dev-ops team should always be aware of these changes as well. Changes to Ingress/Namespace has potentially wide-spreading affects. @mramirez-va @sbair-va @Tay-Sike

File: products/bridg-pathways.conf (hunk 1 above)
Comment (Dearnshaw-VA) [reply products/bridg-pathways.conf]: Yes. Updated.

File: products/bridg-pathways.conf (hunk 2 above)
Comment (Dearnshaw-VA) [reply products/bridg-pathways.conf]: I'm not entirely certain I understand the question. BRIDG is a collection of APIs, this is the first one BRIDG-Pathways, my understanding is ultimately each piece will receive it's own entry in the deployer. I'm willing to follow whatever recommendation you guys make or however DVP does it.

Comment (jgarnhart-va) [reply products/bridg-pathways.conf]: Yes, same crypto key to decrypt the properties as all other apps.

File: products/bridg-pathways.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+metadata:
+  name: bridg-pathways-search-ingress
+  namespace: $DU_NAMESPACE
+  annotations:
+    kubernetes.io/ingress.class: nginx
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /(api/)?bridg-pathways(.*)
```
Comment (jhulbert-va) [inline products/bridg-pathways.yaml]: Is your path `*/pathways/*` or `*/bridg-pathways/*`? The load balancer would route `*/pathways/*` to your bridge applications, but your ingress is looking for `*/bridg-pathways/*`.

Comment (jgarnhart-va) [reply products/bridg-pathways.yaml]: We need it to be bridg-pathways. There will be other bridg apps coming later like bridg-fpds, etc.

Comment (Dearnshaw-VA) [reply products/bridg-pathways.yaml]: Based on that, are there any changes that should be made here @jhulbert-va ?

Comment (mramirez-va) [reply products/bridg-pathways.yaml]: I recommend dropping the `api` part to avoid ingress conflicts with other apps

Comment (Dearnshaw-VA) [reply products/bridg-pathways.yaml]: Update performed.

File: products/bridg-pathways.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+metadata:
+  name: bridg-pathways-search-ingress
+  namespace: $DU_NAMESPACE
+  annotations:
+    kubernetes.io/ingress.class: nginx
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /bridg-pathways(.*)
```
Comment (mramirez-va) [inline products/bridg-pathways.yaml]: Path LGTM

File: products/bridg-pathways.conf [hunk 6]
```diff
@@ -0,0 +1,7 @@
+DU_ARTIFACT=health-apis-bridg-pathways-deployment
+DU_VERSION=1.0.0
+DU_NAMESPACE=bridg-pathways
+DU_DECRYPTION_KEY="$DEPLOYMENT_CRYPTO_KEY"
+DU_HEALTH_CHECK_PATH="/api/pathways/actuator/health"
+DU_LOAD_BALANCER_RULES[1]="*/pathways/*"
```
Comment (mramirez-va) [inline products/bridg-pathways.conf]: This needs to be updated to reflect the ingress path

**Comment by jhulbert-va** [comment]
**DO NOT MERGE**

**Comment by ilaflamme-va** [comment]
@jgarnhart-va this PR is getting stale. Should we close it?

**Pull request closed without merging**